mala/network/predictor.py

```diff
@@ -198,6 +191,10 @@ def _forward_snap_descriptors(
         self, snap_descriptors, local_data_size=None
     ):
         """Forward a scaled tensor of descriptors through the NN."""
+        assert isinstance(
+            snap_descriptors, torch.Tensor
+        ), "snap_descriptors is not a Tensor"
+
```

Comment:
Checking this via an assertion seems like a good idea generally, but just to clarify from my side - is it to be expected that a mismatch here happens often?

Reply:
I don't quite recall why I added that. I guess at some point I provided a numpy array which gave an error. I probably was confused about the fact that some methods expect a Tensor but return a numpy array, which leads users to the assumption that they can provide either.

Anyway, I'm fine with skipping that, also since it is a bit random (there are not too many similar checks in the rest of the code).

Reply:
Got it, yeah I would suggest skipping it for now, maybe at some later point we should introduce such checks in general.

Reply:
OK agreed, I will push a change to this branch then.

Reply:
Changed in 8c78a62a